core/system/inc/page_allocator_config.h

```diff
@@ -42,4 +42,27 @@ typedef uint8_t page_owner_t;
 /* Define a unused value for the page table. */
 #define UVISOR_PAGE_UNUSED ((page_owner_t) (-1))
 
+/** Sets the page bit in the page map array.
+ * @param map   an array of `uint32_t` containing the page map
+ * @param page  the index of the page to be set
```

Comment:
@niklas wouldn't it make more sense to move these defintions into the C file? No need to expose internals .. right ?